team.html

```diff
@@ -10,7 +10,7 @@ <h1>Meet our team</h1>
     <div class="section">
         <h2>Core team</h2>
         <div class="row justify-content-center ">
-            <div class="col-md-8 text-center">
+            <div class="col-md-6 text-center">
```

Comment:
Fixed this width to be consistent with other pages. UX-wise, it's also not recommended to have too many characters like that per line.